azure/biganimal-csp-setup

```diff
@@ -271,10 +269,11 @@ create_or_update_custom_role()
 {
     # get custom role id
     echo "Create or update custom role"
-    role_name="biganimal-poweruser"
+    role_name="biganimal-poweruser-${subscription}"
```

Comment:
Can we use project ID here? Then it will allow us (developers) to do onboard with the same subscription.

Reply:
As I was saying here https://enterprisedb.atlassian.net/browse/UPM-11445?focusedCommentId=367428, I think we can go ahead with this PR which will solve this both in our use case (using the same Subscription with multiple Organizations/Projects) and our Customers use case (in which later they will use a Subscription per Project, rather than a Subscription per Organization like now), because if we onboard a new Project in the same Subscription what will happen is that this process will update the role again with exactly the same content (since the subscription will be the same) and both the previous Project and the new Project will continue to work referring the same Custom Role.But if we prefer to use the Project ID in place of the Subscription ID, we can do that and wait to merge it until we have the `biganimal` CLI Project support.